### PR TITLE
fix: set `svelteFeatures.runes` to `true` by default for Svelte 5

### PR DESCRIPTION
**Pull request by baseballyama**
Follow up of https://github.com/sveltejs/svelte-eslint-parser/pull/536#issuecomment-2173326926

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -383,15 +385,13 @@ For example in `.eslintrc.*`:
       "parser": "svelte-eslint-parser",
       "parserOptions": {
         "parser": "...",
-        "svelteFeatures": { "runes": true },
```
Comment (baseballyama) [inline README.md:386]: I think 95% of users doesn't need to specify this option, so I removed from README.

**Comment by baseballyama** [comment]
Why test is failed~~~

**Comment by baseballyama** [comment]
@ota-meshi Does this PR reflect a shared understanding between us?

**Review by ota-meshi — APPROVED** [review]
Awesome! Thanks!

**Pull request merged**